lua/entities/gmod_wire_lamp.lua

```diff
@@ -10,7 +10,6 @@ end
 
 if CLIENT then
 	local matLight 		= Material( "sprites/light_ignorez" )
-	local matBeam		= Material( "effects/lamp_beam" )
```

Comment:
This local is referenced in some dead commented code. Delete the commented code.